.dockerignore

```diff
@@ -0,0 +1,7 @@
+target/
+cache/
+.dockerignore
```

Comment:
@ChristopherJMiller iirc this is bad news :tm: with some builders, please remove `.dockerignore` from the ignore